src/export-transactions.ts

```diff
@@ -265,6 +271,16 @@ async function tagging(txs: TransactionParsed[]) {
         continue
       }
 
+      if (
```

Comment:
Maybe this could be written as:
```
const isRentalFeeTransaction = tx.type === TransferType.IN && (RENTAL_ADDRESSES.has(tx.from) || RENTAL_ADDRESSES.has(tx.txFrom) || RENTAL_ADDRESSES.has(tx.interactedWith)

if(isRentalFeeTransaction) { ... }
```